### PR TITLE
Remove clone deep from sequence

### PR DESCRIPTION
**Pull request by anthony-murphy**
Was finally able to get a repo of the below issues. Can currently only repo on tinylicious. Will see if i can get a local-server test to hit it in a follow up. Modified todo app to use sequence to debug the issues.

related #3707
related #3956

### CODE REVIEW

File: examples/data-objects/todo/src/Todo/Todo.tsx [hunk 1]
```diff
@@ -84,19 +82,15 @@ export class Todo extends DataObject implements IFluidHTMLView {
         // Create a new todo item
         const component = await TodoItem.getFactory().createChildInstance(this.context, props);
 
-        // Store the id of the component in our ids map so we can reference it later
-        this.todoItemsMap.set(component.handle.absolutePath, component.handle);
+        // Store the handle to the component in the sequence
+        this.todoItems.insert(this.todoItems.getLength(), [component.handle]);
 
         this.emit("todoItemsChanged");
```
Comment (anthony-murphy) [inline examples/data-objects/todo/src/Todo/Todo.tsx:88]:
```suggestion
```

File: packages/tools/webpack-fluid-loader/src/routes.ts [hunk 2]
```diff
@@ -287,7 +287,6 @@ const fluid = (req: express.Request, res: express.Response, baseDir: string, opt
             options,
             document.getElementById("content"))
         .then(() => fluidStarted = true)
-        .catch((error) => console.error(error));
```
Comment (anthony-murphy) [inline packages/tools/webpack-fluid-loader/src/routes.ts:290]: this just makes debugging harder, as i can't just break on all uncaught errors, the errors log regardless

File: packages/dds/sequence/src/sequence.ts [hunk 3]
```diff
@@ -86,7 +82,7 @@ export abstract class SharedSegmentSequence<T extends MergeTree.ISegment>
                 case MergeTree.MergeTreeDeltaType.INSERT:
                     ops.push(MergeTree.createInsertOp(
                         r.position,
-                        cloneDeep(r.segment.toJSONObject())));
+                        r.segment.clone().toJSONObject()));
```
Comment (vladsud) [inline packages/dds/sequence/src/sequence.ts:85]: Add UT to hitting the bug? Relying on todo test is Ok, but UT would be even better!

**Review by vladsud — APPROVED** [review]

File: packages/dds/sequence/src/sequence.ts (hunk 3 above)
Comment (anthony-murphy) [reply packages/dds/sequence/src/sequence.ts:85]: agreed. so far i can't hit the issue in ut's.

**Pull request merged**